bpf/include/linux/byteorder.h

```diff
@@ -1,10 +1,13 @@
-#ifndef _ASM_X86_BYTEORDER_H
-#define _ASM_X86_BYTEORDER_H
+/* SPDX-License-Identifier: GPL-2.0 */
+/* Copyright Authors of Cilium */
```

Comment:
If the commit message here says that it is derived from the Linux kernel repository, then presumably it should include both copyrights...?

Reply:
This file is so trivial that I very much doubt there's matter for a copyright claim at all to be honest. I added the notice just for the sake of having one in the file. But I can add the other copyright notice so that we remain on the safe side.